app/views/registration_wizard/shared/questions/_check_box.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form.govuk_check_boxes_fieldset(question.name, multiple: false, **question.style_options) do %>
+  <% question.body.each do |body_text| %>
+    <p class="govuk-body"><%= body_text %></p>
+  <% end %>
```

Comment:
Oh I like this, would be good to implement in other question types too.

Reply:
It is a bit limited though - we have very little control over where that text is displayed, how it is formatted and only handle simple text - no bullets nor images... I was thinking about passing a block which would allow modification of given question layout, but then we're sort-of entering view-component zone and I ma not sure if that's what we're going for here. Another option would be to write a helper responsible for rendering a question (which could take a block and pass it to the partial)

Reply:
That's fair, it's at least a start of how to get something more flexible in place.